src/sidebar.js

```diff
@@ -200,9 +197,9 @@ class Sidebar extends React.Component {
   }
 
   render() {
-    let sidebarStyle = styles.sidebar,
+    let sidebarStyle = Object.assign({}, styles.sidebar, { zIndex: this.props.zIndex + 1 }),
```

Comment:
I will try to refactor the code a little during my free time. Anyway good work so far! :+1: